devicetypes/smartthings/testing/simulated-refrigerator-temperature-control.src/simulated-refrigerator-temperature-control.groovy

```diff
@@ -24,9 +24,9 @@ metadata {
 		command "setpointDown"
 	}
 
-	tiles {
+	tiles (scale: 1) {
```

Comment:
I guess maybe the scale:2 isn't really a big deal, but the default is `scale: 1` anyways so this wouldn't hurt